packages/tools/examples/segmentSelect/index.ts

```diff
@@ -38,41 +39,48 @@ let renderingEngine;
 const volumeId = 'myVolume';
 const renderingEngineId = 'myRenderingEngine';
 
-const stackViewportId = 'STACK_VIEWPORT';
-const volumeViewportId = 'VOLUME_VIEWPORT';
-
-const stackToolGroupId = 'TOOL_GROUP_STACK';
-const volumeToolGroupId = 'TOOL_GROUP_VOLUME';
-
 // ======== Set up page ======== //
 setTitleAndDescription(
   'Segment Select Tool in Both Stack and Volume Viewport',
```

Comment:
There is some odd behaviour on this - I can select sometimes, but not consistently.  Sometimes it goes to select and sometimes to brush tools.

Reply:
OK, read the code on the select tool.  It is a hover select on delay.  Please update the example to explain to hover over the existing annotations to select one or the other. 
Also, could you add pre-existing annotations for the bottom row so that it is testable?  I think they are supposed to be there, just that I don't see anything.

Reply:
Sure I will add some in the bottom